src/resource.c

```diff
@@ -0,0 +1,18 @@
+#include "resource.h"
+
+#include "res/bluetooth.xbm"
+#include "res/foss-asia-2.xbm"
+
+xbm_t bluetooth = {
+	.bits = bluetooth_bits, 
```

Comment:
nit pick : if you define a struct, you just need to manipulate single pointers in an array